### PR TITLE
#77: Use `project-keeper`.

### LINKED ISSUE #77: Use `project-keeper`
## Acceptance Criteria

1. Project Keeper added to Maven build.

### PR DESCRIPTION
**Pull request by redcatbear**
Closes #77.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Exasol
```
Comment (jakobbraun) [inline LICENSE:3]:
```suggestion
Copyright (c) 2021 Exasol
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
+[![Technical Debt](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Arow-level-security-lua&metric=sqale_index)](https://sonarcloud.io/dashboard?id=com.exasol%3Arow-level-security-lua)
+
+[![Code Smells](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Arow-level-security-lua&metric=code_smells)](https://sonarcloud.io/dashboard?id=com.exasol%3Arow-level-security-lua)
+[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Arow-level-security-lua&metric=coverage)](https://sonarcloud.io/dashboard?id=com.exasol%3Arow-level-security-lua)
+[![Duplicated Lines (%)](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Arow-level-security-lua&metric=duplicated_lines_density)](https://sonarcloud.io/dashboard?id=com.exasol%3Arow-level-security-lua)
+[![Lines of Code](https://sonarcloud.io/api/project_badges/measure?project=com.exasol%3Arow-level-security-lua&metric=ncloc)](https://sonarcloud.io/dashboard?id=com.exasol%3Arow-level-security-lua)
+-->
```
Comment (jakobbraun) [inline README.md:19]: ;-) :+1 nice quick fix

File: pom.xml [hunk 3]
```diff
@@ -159,7 +177,17 @@
...
+        <!-- Set the highest log level for coverage testing, so that we have a chance to reach branches
+          <property>	                      in the logging lambdas too. -->
+        <argLine>-Djava.util.logging.config.file=src/test/resources/logging.properties ${argLine}</argLine>
+        <includes>
+            <include>**IT.java</include>
+        </includes>
+        <systemPropertyVariables>
+            
+        </systemPropertyVariables>
+    </configuration></plugin>
```
Comment (jakobbraun) [inline pom.xml:190]: biite einem reformatieren

**Review by jakobbraun — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (redcatbear) [reply pom.xml]: Done.

File: LICENSE (hunk 1 above)
Comment (redcatbear) [reply LICENSE:3]: Done.

File: src/test/java/com/exasol/rls/administration/AbstractAdminScriptIT.java [hunk 4]
```diff
@@ -54,14 +55,31 @@ protected static void initialize(final ExasolContainer<? extends ExasolContainer
...
+    public AbstractAdminScriptIT() {
+        try {
+            this.connection = EXASOL.createConnection("");
+        } catch (NoDriverFoundException | SQLException exception) {
+            throw new RuntimeException(exception);
+        }
+    }
+
+    @Override
+    protected void finalize() {
```
Comment (jakobbraun) [inline src/test/java/com/exasol/rls/administration/AbstractAdminScriptIT.java:67]: Finalize is deprecated. Can't you use AfterAll or closable here?

**Review by jakobbraun — CHANGES_REQUESTED** [review]

File: src/test/java/com/exasol/rls/administration/AbstractAdminScriptIT.java (hunk 4 above)
Comment (redcatbear) [reply src/test/java/com/exasol/rls/administration/AbstractAdminScriptIT.java]: I reduced all connections to a single static one. The instance-based option was an artifact of the Java variant anyway where three different connection types needed to be tested. This is not relevant in the Lua variant.

**Review by jakobbraun — APPROVED** [review]

**Pull request merged**